src/VisualStudio/CSharp/Impl/Options/Formatting/CodeStylePage.cs

```diff
@@ -11,6 +11,7 @@
 using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.CSharp.CodeStyle;
 using Microsoft.CodeAnalysis.CSharp.Formatting;
+using Microsoft.CodeAnalysis.ExternalAccess.EditorConfig;
```

Comment:
>using Microsoft.CodeAnalysis.ExternalAccess.EditorConfig [](http://example.com/codeflow?start=0&length=56)

We shouldn't have our own product code using ExternalAccess layer types. Those should be only used by external code.

Reply:
Nothing references ExternalAccess anymore